news/util.py

```diff
@@ -0,0 +1,54 @@
+"""Utility functions for interacting with OCW News"""
+import re
+
+from bs4 import BeautifulSoup
+
+
+def get_original_image_link(url):
+    """
+    Parse the image URL to cut off the 150x150 from the RSS feed
+
+    Args:
+        url (str): The image URL
+
+    Returns:
+        str: The updated URL if it matches the regex, else the original URL
+    """
+    match = re.match(r"^(?P<prefix>.+)-150x150\.(?P<suffix>.+)$", url)
+    if match:
+        return f"{match.group('prefix')}.{match.group('suffix')}"
+    return url
+
+
+def serialize_item(item):
+    """
+    Serialize an RSS feed item
+
+    Args:
+        item (xml.etree.elementTree.Element): An <item>
+
+    Returns:
+        dict: A dictionary representation of the RSS item
+    """
+    obj = {child.tag: {"text": child.text, **child.attrib} for child in item}
+    description = obj["description"]
```

Comment:
Some of the item attributes look a bit funky in JSON:

{http://purl.org/dc/elements/1.1/}creator
{http://purl.org/rss/1.0/modules/content/}encoded
{http://wellformedweb.org/CommentAPI/}commentRss
{http://purl.org/rss/1.0/modules/slash/}comments

Maybe the curly braces could be removed for the JSON output?